x-pack/plugins/search_indices/public/components/indices/details_page.tsx

```diff
@@ -232,33 +268,15 @@ export const SearchIndexDetailsPage = () => {
                     >
                       <FormattedMessage
                         id="xpack.searchIndices.indexAction.ApiReferenceButtonLabel"
-                        defaultMessage="{buttonLabel}"
-                        values={{
-                          buttonLabel: isDocumentsLoading ? 'Loading' : 'API Reference',
-                        }}
+                        defaultMessage="API Reference"
                       />
                     </EuiButtonEmpty>
-                  ) : (
-                    <EuiButtonEmpty
-                      isLoading={isDocumentsLoading}
-                      iconType="launch"
-                      data-test-subj="useInPlaygroundLink"
-                      onClick={navigateToPlayground}
-                    >
-                      <FormattedMessage
-                        id="xpack.searchIndices.indexAction.useInPlaygroundButtonLabel"
-                        defaultMessage="{buttonLabel}"
-                        values={{
-                          buttonLabel: isDocumentsLoading ? 'Loading' : 'Use in Playground',
-                        }}
-                      />
-                    </EuiButtonEmpty>
-                  )}
-                </EuiFlexItem>
+                  </EuiFlexItem>
+                )}
                 <EuiFlexItem>
                   <SearchIndexDetailsPageMenuItemPopover
                     handleDeleteIndexModal={handleDeleteIndexModal}
-                    navigateToPlayground={navigateToPlayground}
+                    showApiReference={isDocumentsExists}
```

Comment:
nitpick: hasDocuments
```suggestion
                    showApiReference={hasDocuments}
```